### PR TITLE
Add template for Reach Us page

### LINKED ISSUE #109: Add template for Reach Us page

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #109

### CODE REVIEW

File: festflow/templates/festflow/base.html [hunk 1]
```diff
@@ -22,11 +22,14 @@
 <body>
     {% navbar %}
     {% block body_contents %}{% endblock %}
+</body>
 
-    <!-- Jquery minified js -->
-    <script type="text/javascript" src='{% static "js/jquery.min.js"%}'></script>
+<!-- Jquery minified js -->
```
Comment (ghoshbishakh) [inline festflow/templates/festflow/base.html]: why did you make this change? This is not correct. Put the scripts back to the end of the body and not outside it.

Comment (akshatnitd) [inline festflow/templates/festflow/base.html]: Doing the change :+1:

**Comment by akshatnitd** [comment]
@ghoshbishakh Corrected it :+1:

File: festflow/templates/festflow/reachus.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+{% endblock %}
+
+{% block body_contents %}
+
+<div class="container">
+    <div class="jumbotron">
+        <h1 id='link'>Reach Us</h1>
+        <div class="dash"></div>
+        <br />
+        <br />
```
Comment (nkprince007) [inline festflow/templates/festflow/reachus.html]: Please don't use line breaks unless absolutely necessary.

File: festflow/templates/festflow/reachus.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            center: college
+        });
+        var marker = new google.maps.Marker({
+            position: college,
+            map: map
+        });
+    }
+</script>
+<script async defer
+src="https://maps.googleapis.com/maps/api/js?key={{ google_api_key }}&callback=initMap">
```
Comment (nkprince007) [inline festflow/templates/festflow/reachus.html]: add a setting for fetching google_api_key. 😉

File: festflow/templates/festflow/reachus.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
+{% extends "festflow/base.html" %}
+{% load static %}
+
+{% block title %} REACH US {% endblock %}
+
+{% block extra_head %}
+
```
Comment (nkprince007) [inline festflow/templates/festflow/reachus.html]: why not use the navbar here too?

**Comment by ghoshbishakh** [comment]
@nkprince007 Lots of changes are needed for this view. Let me merge this particular PR. Ill be opening issues for the rest of the changes.

File: festflow/templates/festflow/reachus.html (hunk 4 above)
Comment (akshatnitd) [inline festflow/templates/festflow/reachus.html]: Navbar is already included in the base.html. So, its already added!

File: festflow/templates/festflow/reachus.html (hunk 3 above)
Comment (akshatnitd) [inline festflow/templates/festflow/reachus.html]: Isn't the setting already present? The API key is passed as the context, and I have used that value.

File: festflow/templates/festflow/reachus.html (hunk 4 above)
Comment (nkprince007) [inline festflow/templates/festflow/reachus.html]: Oh. Yeah. Sorry.

**Pull request merged**